packages/appstore/src/components/modals/real-wallets-upgrade/real-wallets-upgrade.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { DesktopWrapper, MobileDialog, MobileWrapper, Modal, Button } from '@deriv/components';
+import { useWalletMigration } from '@deriv/hooks';
+import { ContentFlag } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { WalletsIntro } from './components/wallets-intro/wallets-intro';
+import ReadyToUpgradeWallets from './components/ready-to-upgrade-wallets';
+import './real-wallets-upgrade.scss';
+import WalletLinkingStep from './wallet-linking-step/wallet-linking-step';
+import getMockWalletMigrationResponse from '../../../constants/mock_wallet_migration_response';
```

Comment:
I see that in other files the imports arent ordered properly, i know that different squads have different ways they order the imports, but currently in p2p we are doing something like this:

- React
- Third Party Libraries
- @deriv imports
- Other packages Components
- Local Components
- Constants, Utils, local store files
- Components relative to current component
- style file (if applicable)

<img width="663" alt="Screenshot 2023-08-17 at 10 35 08 AM" src="https://github.com/binary-com/deriv-app/assets/103412909/eaac8997-b42a-4c87-8624-0419a2f6a93a">